tests/napalm_connection/connection_info_in_args.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Get facts
+  hosts: all
+  connection: local                       # code is run locally
+  gather_facts: no                        # don't gather facts
+  tasks:
+    - block:
+      - name: get facts from device
+        napalm_get_facts:                   # NAPALM plugin
+          dev_os: "{{ os }}"
+          optional_args:
+            path: "{{ playbook_dir }}/mocked"
+            profile: "{{ profile }}"
+          filter: ['facts']
+        register: napalm_facts             # store information here
+      rescue:
+        # This is expected to fail in ansible < 2.3
```

Comment:
I think we should drop Ansible 2.2 support. Thoughts @itdependsnetworks and @ogenstad

Reply:
2.2 has been my gold standard until thie fix the cannot open shell logging issues (which should be 2.5). Would prefer to keep it for a least a few months.

Reply:
Okay, that is fine with me as long as things don't break materially.

Reply:
Yeah, fine by me as well.